engine/src/main/java/org/terasology/engine/config/flexible/AutoConfigManager.java

```diff
@@ -84,7 +84,7 @@ public void loadConfigsIn(Context context) {
         try (InputStream inputStream = Files.newInputStream(configPath, StandardOpenOption.READ)) {
             T loadedConfig = (T) serializer.deserialize(TypeInfo.of(configClass), inputStream).get();
             mergeConfig(configClass, loadedConfig, config);
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
Hm, I think it is usually a "bad practice" to just capture `Exception`. Do we know what exception can occur here? What else other than `IOException` is thrown here?